packages/genesys-spark-components/scripts/stencil-wrapper.ts

```diff
@@ -11,35 +11,28 @@ import { defineCustomElements } from '../dist/loader';
 // Value templated out during build process (see scripts/wrap-stencil.js)
 export const COMPONENT_ASSETS_PATH = '{{component_assets_path}}';
 
-// Default domain to load assets from
-const DEFAULT_DOMAIN = 'app.mypurecloud.com';
-
-// List of Genesys UI domains
 const DOMAIN_LIST = [
-  'apne2.pure.cloud',
-  'apne3.pure.cloud',
-  'aps1.pure.cloud',
-  'cac1.pure.cloud',
-  'euc2.pure.cloud',
-  'euw2.pure.cloud',
-  'inindca.com',
-  'inintca.com',
-  'mec1.pure.cloud',
-  'mypurecloud.com',
-  'mypurecloud.com.au',
-  'mypurecloud.de',
-  'mypurecloud.ie',
-  'mypurecloud.jp',
-  'sae1.pure.cloud',
-  'use2.maximus-pure.cloud',
-  // 'use2.us-gov-pure.cloud', Assets are not currently deployed to FedRAMP and should fallback to the default domain
-  'usw2.pure.cloud'
+  '.inindca.com',
+  '.dev-pure.cloud',
+
+  '.inintca.com',
+  '.test-pure.cloud',
+
+  '.mypurecloud.com',
+  '.mypurecloud.com.au',
+  '.mypurecloud.de',
+  '.mypurecloud.ie',
+  '.mypurecloud.jp',
+
+  '.pure.cloud',
+  '.maximus-pure.cloud'
+  // '.use2.us-gov-pure.cloud', Assets are not currently deployed to FedRAMP and should fallback to the default domain
```

Comment:
Rather than hard-coding all these could you add a dependency on the public service discovery package and then only hard-code the domains for lowers? I used the internal service discovery package (which includes lowers), but I could show you what I did in my package if it helps.

Reply:
I will look at this in a follow up PR